mne/io/diff.py

```diff
@@ -1,40 +0,0 @@
-# Author: Alexandre Gramfort <alexandre.gramfort@inria.fr>
-#
-# License: BSD Style.
-
-import numpy as np
-
-from ..utils import logger, verbose
-
-
-@verbose
-def is_equal(first, second, verbose=None):
```

Comment:
This was intentionally removed. I had never seen it before, it was never used anywhere AFAIK